libs/ledger-live-common/src/families/casper/bridge.integration.test.ts

```diff
@@ -0,0 +1,151 @@
+      => 11010000142c000080fa010080020000800000000000000000
+      <= 029644cc1ec96ea3faf36d483b7bde8c85393b9b34e602f184d8b559b526e474ed30323032393634344363314563393665413346414633364434383362374264653863383533393342396233344536303246313834643862353539623532364534373445649000
+      => 11010000142c000080fa010080030000800000000000000000
+      <= 02488e51a828423e59a45261c63c473d59419a8d7d2dc29fbadd9e8fee16d3988330323032343838453531613832383432334535394134353236314336336334373364353934313941386437443244633239666241644439453866456531366433393838339000
+      `,
+    },
+  ],
+  accounts: [
+    {
+      raw: {
+        id: `js:2:casper:${SEED_IDENTIFIER}:casper_wallet`,
+        seedIdentifier: SEED_IDENTIFIER,
+        name: "Casper 1",
+        derivationMode: "casper_wallet" as const,
+        index: 0,
+        freshAddress: SEED_IDENTIFIER,
+        freshAddressPath: "44'/506'/0'/0/1",
+        freshAddresses: [],
+        blockHeight: 0,
+        operations: [],
+        pendingOperations: [],
+        currencyId: "casper",
+        unitMagnitude: 9,
+        lastSyncDate: "",
+        balance: "1000",
+      },
+      transactions: [
+        {
+          name: "Not a valid address",
+          transaction: fromTransactionRaw({
+            family: "casper",
+            recipient: "novalidaddress",
+            amount: "1000",
+            deploy: null,
+          }),
+          expectedStatus: {
+            errors: {
+              recipient: new InvalidAddress(),
+            },
+            warnings: {},
+          },
+        },
+        {
+          name: "Not enough balance",
+          transaction: fromTransactionRaw({
+            family: "casper",
+            recipient: ACCOUNT_2,
+            amount: (300 * 1e9).toString(),
+            deploy: null,
+          }),
+          expectedStatus: {
+            errors: {
+              amount: new NotEnoughBalance(),
+            },
+            warnings: {},
+          },
+        },
+        {
+          name: "Amount Required",
+          transaction: fromTransactionRaw({
+            family: "casper",
+            recipient: ACCOUNT_2,
+            amount: "0",
+            deploy: null,
+          }),
+          expectedStatus: {
+            errors: {
+              amount: new AmountRequired(),
+            },
+            warnings: {},
+          },
+        },
+        {
+          name: "Minimum Amount Required",
+          transaction: fromTransactionRaw({
+            family: "casper",
+            recipient: ACCOUNT_2,
+            amount: "1",
+            deploy: null,
+          }),
+
+          expectedStatus: {
+            errors: {
+              amount: new AmountRequired(
+                `Minimum CSPR to transfer is ${motesToCSPR(
+                  MINIMUM_VALID_AMOUNT
+                ).toNumber()} CSPR`
```

Comment:
Could we convert this into some function and call it from wherever we need? So we avoid DRY.

Reply:
Yeah, about this. A function is perfect, should do it. I created a custom error in past called `InvalidMinimumAmount` but then figured that we will need to translate it to other languages, etc. just like other errors. 

Now if we use the error `AmountRequired` with custom message, the message doesn't appear in the UI. Which is not good... What do you think we should do? Does creating new error `InvalidMinimumAmount` make the most sense here?

Reply:
Done added `InvalidMinimumAmount` custom error. Note that we do not provide transalations for this error class.